content/03.results.md

```diff
@@ -31,20 +31,20 @@ We used the same experimental setup as in our other binary prediction tasks to t
 In this setting we found that while the models all performed similarly, the nonlinear models tended to have a slight edge over the linear one.
 
 ![
-Performance of models across four classification tasks
+Performance of models across four classification tasks. In each panel, confidence intervals show mean and standard error across 5 folds of studywise cross-validation.
```

Comment:
Is this true? I'm using the default `geom_smooth` function in plotnine which is loess curve + 95% confidence interval. I think that's a local regression line and it's confidence, not the true mean and SE, but I could be wrong

Reply:
Ahh, yeah, I think you're right!